guide/getting-started/getting-started.md

```diff
@@ -57,6 +69,11 @@ Learn the basics behind the technologies that make bitcoin work, from what a bit
 
 ---
 
+### [Bitcoin design principles]({{ '/guide/getting-started/principles/' | relative_url }})
+
+Core bitcoin design principles recommended by the Bitcoin Design Community.
+
+---
 ### [Software overview]({{ '/guide/getting-started/software/' | relative_url }})
```

Comment:
Is software or hardware suppose to be first in the order?

Reply:
Software should be first. The menu was wrong. I've updated it now.